Only in ../dropcenter: .git
Only in ../dropcenter: .gitignore
diff -ur ../dropcenter/header.php sources/header.php
--- ../dropcenter/header.php	2015-03-14 16:01:45.805821759 +0100
+++ sources/header.php	2015-03-14 18:20:28.718110945 +0100
@@ -62,8 +62,9 @@
 $tpl->assign('DC_WEBSITE',DC_WEBSITE);//Site du programme
 
 if(file_exists('./'.DCFOLDER.USERFILE)){
-		$user = (isset($_SESSION['user']) && trim($_SESSION['user'])!='' && $_SESSION['user']!=null ?@unserialize($_SESSION['user']):null);
-		$user = ($user?$user:null);
+		$user = parseUsers('./')[0];
+		$_SESSION['user'] = serialize($user);
+		$_SESSION['tpmToken'] = sha1(time().rand(0,100));
 		$tpl->assign('user',$user);
 		$_ = getLang();
 }else{
@@ -71,4 +72,4 @@
 	header('location: install.php');
 	}
 }
-?>
\ No newline at end of file
+?>
diff -ur ../dropcenter/php/config.php sources/php/config.php
--- ../dropcenter/php/config.php	2015-03-14 16:01:45.805821759 +0100
+++ sources/php/config.php	2015-03-17 00:52:07.006904679 +0100
@@ -1,14 +1,14 @@
 <?php
 
 
-define('DC_TITLE','DropCenter');//Titre du dropCenter
-define('DC_DESCRIPTION','File drop center kiss n\'fun');//description du dropCenter
+define('DC_TITLE','PirateBox');//Titre du dropCenter
+define('DC_DESCRIPTION','Upload and download freely');//description du dropCenter
 define('DC_LOGO','img/logo.png');//logo central du dropCenter (Si rien n'est mis, le logo par défaut apparaît)
 define('DC_LICENCE','CC BY NC SA');//License de votre dropCenter
 define('UPLOAD_FOLDER','uploads/');//chemin vers le dossier d'upload (ne pas oublier de mettre les droits d'écriture sur ce dossier)
 define('NAME_LIMIT',25);//Nombre maximal de caractères affichés pour les fichiers
 define('NAME_LIMIT_BORDER','..');//Les caractères qui s'afficheront pour signifier qu'un nom est raccourci
-define('MAX_SIZE',1000);//Taille maximale authorisée par fichier en Mo (Pensez a configurer post_max_size et upload_max_size dans le fichier php.ini de votre serveur si vous voulez uploader de gros fichiers).
+define('MAX_SIZE',1024);//Taille maximale authorisée par fichier en Mo (Pensez a configurer post_max_size et upload_max_size dans le fichier php.ini de votre serveur si vous voulez uploader de gros fichiers).
 define('FORBIDEN_FORMAT','exe,php,sh,bin,htaccess,htm,html,asp');//Les extensions interdites à l'exécution séparées par des virgules (les fichiers seront bien envoyés mais un .txt sera rajouté à l'extension afin d'empêcher les utilisateurs d'exécuter leurs fichiers sur le serveur
 define('AVATAR_DEFAULT','defaultAvatar.png'); //chemin de l'avatar par défaut
 define('AVATARFOLDER',UPLOAD_FOLDER.'avatars/'); //dossier contenant les avatars
@@ -17,8 +17,8 @@
 define('READ_FOR_ANONYMOUS',false);// Définit si les visiteurs non authentifiés peuvent lire le contenu du dropCenter (true = lecture possible, false = lecture interdite)
 define('DC_LANG','fr - Francais');//Définit la langue par défaut
 define('DIR_LANG','lang/');//Dossier des fichiers de langue
-define('MAIL',true);//Autorise les notifications par e-mail
-define('DISPLAY_DOTFILES',false);//Affiche ou non les dossiers/fichiers commençant par un point
+define('MAIL',false);//Autorise les notifications par e-mail
+define('DISPLAY_DOTFILES',true);//Affiche ou non les dossiers/fichiers commençant par un point
 define('DISPLAY_UPDATE',false);//Activer la vérification des mises-à-jour
 define('DISPLAY_AVATAR_FOLDER',false);//Afficher le dossier des avatars
 define('DEFAULT_THEME','UnderBlack');//Afficher le dossier des avatars
diff -ur ../dropcenter/tpl/UnderBlack/footer.html sources/tpl/UnderBlack/footer.html
--- ../dropcenter/tpl/UnderBlack/footer.html	2015-03-14 16:01:45.809821797 +0100
+++ sources/tpl/UnderBlack/footer.html	2015-03-15 21:49:44.413627821 +0100
@@ -18,13 +18,6 @@
 	{if="$FORTUNE"}<br/><br/>{function="chuckQuote()"}<br/><br/>{/if}
 	
 
-	{if="isset($user)"}
-	<a class="rssFeed tooltips" target="_blank"
-		href="php/action.php?action=login&token={function="getToken($user)"}" alt="<?php t("Flux RSS");?>"
-		title="{function="t("Abonnez vous au flux rss pour suivre les evenements du DropCenter")"}"><i class="icon-rss"></i>{function="t("Flux RSS")"}
-	</a>
-	{/if}
-
 </footer>
 
 
@@ -53,4 +46,4 @@
 {/if}
 
 </body>
-</html>
\ No newline at end of file
+</html>
Binary files ../dropcenter/tpl/UnderBlack/img/logo.png and sources/tpl/UnderBlack/img/logo.png differ
diff -ur ../dropcenter/tpl/UnderBlack/index.html sources/tpl/UnderBlack/index.html
--- ../dropcenter/tpl/UnderBlack/index.html	2015-03-14 16:01:45.813821833 +0100
+++ sources/tpl/UnderBlack/index.html	2015-03-14 18:21:24.470657608 +0100
@@ -21,7 +21,7 @@
 			
 			<div class="menuIcon">
 				<span onclick="$('.folderNameBloc').fadeToggle(150);" title="{function="t("Nouveau dossier")"}" class="tooltips">
-				<i class="icon-folder"></i></span><span><i onclick="$('#paramsBloc').fadeToggle()" title="{function="t("Parametres") "}" class="icon-setting tooltips"></i></span><!--<span><i onclick="window.location='php/action.php?action=backup';" class="icon-backup tooltips" title="{function="t("Sauvegarde")"}"></i></span>--><span><i onclick="$('#usersBloc').fadeToggle()" title="{function="t("Comptes")"}" class="icon-user tooltips"></i></span><span><i onclick="window.location='php/action.php?action=logout';" class="icon-cross tooltips" title="<figure class='avatar'><img src='php/action.php?action=openFile&file={$user->avatar}'/></figure>{function="t('Deconnexion')"}<br/>{$user->login}"></i></span>
+				<i class="icon-folder"></i></span>
 			</div>
 
 
diff -ur ../dropcenter/tpl/UnderBlack/js/main.js sources/tpl/UnderBlack/js/main.js
--- ../dropcenter/tpl/UnderBlack/js/main.js	2015-03-14 16:01:45.821821907 +0100
+++ sources/tpl/UnderBlack/js/main.js	2015-03-17 01:01:03.124240012 +0100
@@ -302,7 +302,7 @@
 						'<div class="fileUrl">'+stripslashes(file.url)+'</div>'+
 						'<span title="'+stripslashes(file.toolTipName)+'" class="imageHolder'+(file.published?' filePublished':'')+'"><div onclick="deleteFile(this)" class="deleteFile">x</div>'+
 							
-							'<div onclick="focusFile(this)"  ondblclick="openFile(this)">'+
+							'<div onclick="openFile(this)">'+
 
 							'<img width="48px" height="48px"  src="'+ext+'"/>'+
 							'<ul>'+
@@ -312,18 +312,7 @@
 							'</ul>'+
 
 							'</div>'+
-							'<ul><li class="fileOption">+Options</li></ul>'+
-
 							'<span ondblclick="renameFile(this)\" title="'+stripslashes(file.name)+'" alt="'+stripslashes(file.name)+'"  class="fileName">'+stripslashes(file.shortname)+'</span>'+
-						'<div class="addOptions">'+
-						'<ul>'+
-						'<li onclick="$(\'.directLink\',$(this).parent().parent().parent()).fadeToggle(200).select();" alt="Copier le lien direct" title="Copier le lien direct" class="optionUrl"></li>'+
-						//'<li alt="Envoyer par mail" title="Envoyer par mail" class="optionShare"></li>'+
-						//'<li alt="Editer la source" title="Editer la source" class="optionEdit"></li>'+
-						'<li onclick="zipFile(this)" alt="T&eacute;l&eacute;charger le fichier compressé" title="T&eacute;l&eacute;charger le fichier compressé" class="optionZip"></li>'+
-						'<li onclick="'+(file.published?'un':'')+'publishFile(this)" title="Public/Privé" class="optionDropbox"></li>'+
-						'</ul><div class="clear"></div></div>'+
-						'<textarea type="text" class="directLink">'+stripslashes(file.absoluteUrl)+'</textarea>'+
 						'</span>'+
 						'</div>'; 
 
@@ -628,4 +617,4 @@
     
     if(is_list) return '[' + json + ']';//Return numerical JSON
     return '{' + json + '}';//Return associative JSON
-}
\ No newline at end of file
+}
Only in ../dropcenter/uploads: .htaccess
